installation-usage.md

```diff
@@ -165,14 +165,16 @@ A helm chart is also available for installation. The chart is attached to each
 [GitHub release](https://github.com/kubernetes-sigs/security-profiles-operator/releases)
 as an artifact, and can be installed by executing the following shell commands:
 
+You may also specify a different target namespace with `--namespace mynamespace` or `--namespace mynamespace --create-namespace` if it still doesn't exist.
 ```shell
 # Install cert-manager if it is not already installed (TODO: The helm
 # chart might do this one day - see issue 1062 for details):
 kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.0/cert-manager.yaml
```

Comment:
@saschagrunert @pjbgf  should this be updated to [v1.11.0](https://github.com/cert-manager/cert-manager/releases/tag/v1.11.0)?

Reply:
I would that on a separate PR.